optimize/util/optimize-commons/src/main/resources/service-config.yaml

```
@@ -147,6 +147,8 @@ zeebe:
   maxImportPageSize: ${CAMUNDA_OPTIMIZE_ZEEBE_MAX_IMPORT_PAGE_SIZE:200}
   # Determines whether Optimize should convert and store object variables from Zeebe
   includeObjectVariableValue: ${CAMUNDA_OPTIMIZE_ZEEBE_INCLUDE_OBJECT_VARIABLE:true}
+  # Indicates whether variable import is enabled
+  variablesImportEnabled: ${CAMUNDA_OPTIMIZE_ZEEBE_VARIABLES_IMPORT_ENABLED:true}
```

```suggestion
  variableImportEnabled: ${CAMUNDA_OPTIMIZE_ZEEBE_VARIABLE_IMPORT_ENABLED:true}
```
I think we normally refer it to variable import (not pluralised). Not a huge difference, but let's aim for consistency with it

---

That is renamed